build/compile-ts.js

```diff
@@ -5,6 +5,7 @@
  */
 
 var gulp = require("gulp");
+var plumber = require("gulp-plumber");
```

Comment:
What issue is `gulp-plumber` addressing?

Reply:
While running `gulp serve` the compile Typescript task can throw an error into the pipe when you have a syntax error, so plumber monkey patches NodeJS pipe to not kill the stream when an error is thrown.